tensorflow/core/kernels/fused_batch_norm_op.cc

```diff
@@ -62,6 +71,121 @@ struct CastIfNecessary<true, Y, X, T> {
   }
 };
 
+template <typename U, typename T>
+DeviceMemory<U> CastDeviceMemory(Tensor* tensor) {
+  return DeviceMemory<U>::MakeFromByteSize(
+      tensor->template flat<T>().data(),
+      tensor->template flat<T>().size() * sizeof(T));
+}
+
+inline se::port::Status ToExecutorStatus(const Status& s) {
+  return s.ok() ? se::port::Status::OK()
```

Comment:
Why converting? I thought they're of the same type: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/stream_executor/lib/status.h#L29